### PR TITLE
Rename menu and route for the elastic agent configuration

### PR DESCRIPTION
**Pull request by bdpiparva**
- Renamed menu to `Elastic Agent Configurations`
- Rename page header
- Changed spa route from `/go/admin/elastic_profiles` to `/go/admin/elastic_agent_configurations`
- Added redirect from `/go/admin/elastic_profiles` to `/go/admin/elastic_agent_configurations`(in case of bookmarks)

### CODE REVIEW

File: server/webapp/WEB-INF/urlrewrite.xml [hunk 1]
```diff
@@ -389,6 +389,12 @@
     <to last="true">/spark/admin/plugins</to>
   </rule>
 
+  <rule>
+    <name>Elastic Agent Configurations SPA</name>
+    <from>^/admin/elastic_profiles(/?)$</from>
+    <to last="true">/spark/admin/elastic_agent_configurations</to>
```
Comment (ketan) [inline server/webapp/WEB-INF/urlrewrite.xml]: This looks like an internal rewrite, not a redirect.

Comment (ketan) [reply server/webapp/WEB-INF/urlrewrite.xml]: See http://tuckey.org/urlrewrite/manual/4.0/#to

`<to last="true" type="temporary-redirect" />`

Comment (bdpiparva) [reply server/webapp/WEB-INF/urlrewrite.xml]: Sure will update this

Comment (GaneshSPatil) [reply server/webapp/WEB-INF/urlrewrite.xml]: @ketan -- just  a question regarding `permanent-redirect` (301) vs `temporary-redirect` (302).

As the old URL is moved permanently and will be removed (in near future), should we use `permanent-redirect`?

**Comment by ketan** [comment]
I'd prefer a temporary redirect, allows us to reuse the old url for
something else. A permanent redirect is cached by the browser, not sure how
long for.

**Pull request merged**